<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>styleManagerCubeViewer</class>
 <widget class="QDialog" name="styleManagerCubeViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>82</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>styleManagerCubeViewer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="rowWrapPolicy">
        <enum>QFormLayout::WrapAllRows</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="stretchLabel">
         <property name="text">
          <string>Stretch</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="stretchComboBox">
         <item>
          <property name="text">
           <string>Linear</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sqrt</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Log</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QFormLayout" name="formLayout_4">
       <property name="rowWrapPolicy">
        <enum>QFormLayout::WrapAllRows</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="scaleLabel">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="scaleComboBox">
         <item>
          <property name="text">
           <string>MinMax</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ZScale</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout">
       <property name="rowWrapPolicy">
        <enum>QFormLayout::WrapAllRows</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="colorLabel">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="colorComboBox">
         <item>
          <property name="text">
           <string>Gray</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Accent</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Heat</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rainbow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CoolWarm</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
